.github/workflows/default_branch_verification.yml

```diff
@@ -11,3 +11,9 @@ jobs:
     with:
       command: ./gradlew build
     secrets: inherit
+
+  app-distribution:
```

Comment:
minor: would be nice to have more explanatory name, e.g. `upload-debug-apk` or something similar, to clearly point what action will be executed

Reply:
Applied in PR https://github.com/intive/Picover/pull/202